pallets/liquidity-pools-gateway/src/lib.rs

```diff
@@ -162,11 +164,48 @@ pub mod pallet {
 			hook_address: [u8; 20],
 		},
 
```

Comment:
@hieronx the following events should roughly match the ones that you described:

```
event ProcessMessage(bytes message, address adapter);
event ProcessProof(bytes32 messageHash, address adapter);
event ExecuteMessage(bytes message, address adapter); // liquidityPools.IncomingMessage now
event SendMessage(bytes message); // liquidityPoolsGateway.MessageSubmitted now
event RecoverMessage(address adapter, bytes message);
event RecoverProof(address adapter, bytes32 messageHash);
```

We have a different naming convention for events and I wouldn't change that tbh, unless we change all of them...

Please note that:
- we're only using message hashes since using the actual bytes would require `BoundedVec`s.
- the adapter address is not something that we have access to at this point, I'm not sure how easy it would be to grab those for each router. Maybe @lemunozm @wischli can help.

Reply:
Sounds great, thanks for adapting this!

I just realized, should we add `Domain` to `InboundMessageProcessed`, `InboundProofProcessed`, and `OutboundMessageSent`? 

This is not included on the Solidity side  since it is always Centrifuge there. But here it seems we need it.

Reply:
>I just realized, should we add Domain to InboundMessageProcessed, InboundProofProcessed, and OutboundMessageSent?

Sure.

Reply:
Added `domain_address` to all inbound / outbound events. Note that for outbound, that address is the `T::Sender`.